README.md

```diff
@@ -16,6 +16,7 @@
 </a>
 
 <br/><br/>
+
```

Comment:
Revert unnecessary changes.

Reply:
done

Reply:
There's still a change that should be reverted.